src/setupTests.js

```diff
@@ -0,0 +1,33 @@
+import render, { shallowRender } from 'preact-render-to-json';
+
+/**
+ * Remove __self and __source from the list of attributes/properties.
+ * babel-preset-react adds these for use with the React DevTools
+ */
+function removeDevAttributes(vnode) {
```

Comment:
`babel-preset-react` adds `__self` and `__source` as `props` to every `Component`.

We need this small patch to make them go away and have reliable snapshot testing.

See https://github.com/developit/preact-render-to-string/issues/43#issuecomment-328118525 for more info.